pkgs/applications/networking/mailreaders/notmuch/mutt.nix

```diff
@@ -38,8 +38,8 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     inherit version;
     description = "Mutt support for notmuch";
-    homepage    = http://notmuchmua.org/;
-    license     = with licenses; mit;
+    homepage    = https://notmuchmail.org/;
```

Comment:
Can you keep the field alignment as it was? Other than that, I think this is good to go.